shared-bindings/busio/UART.c

```diff
@@ -345,18 +340,16 @@ STATIC mp_obj_t busio_uart_obj_reset_input_buffer(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(busio_uart_reset_input_buffer_obj, busio_uart_obj_reset_input_buffer);
 
-//| .. class:: busio.UART.Parity()
-//|
-//|     Enum-like class to define the parity used to verify correct data transfer.
-//|
-//|     .. data:: ODD
-//|
-//|       Total number of ones should be odd.
-//|
-//|     .. data:: EVEN
-//|
-//|       Total number of ones should be even.
-//|
+//|class busio:
```

Comment:
I think this would be `class Parity` and ODD and EVEN are class members defined outside of `__init__`.